LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTemplateMods.uc

```diff
@@ -1176,6 +1177,12 @@ function ModifyAbilitiesGeneral(X2AbilityTemplate Template, int Difficulty)
 		}
 	}
 
+	if(Template.DataName == 'RapidFire') 
```

Comment:
For future reference, please put spaces after `if`, `for`, `switch`, etc. I know there are plenty of inconsistencies in the current code, but want to make new stuff follow a standard. I'll fix it locally.